lib/generators/solidus_braintree/update/update_generator.rb

```diff
@@ -0,0 +1,23 @@
+# frozen_string_literal: true
+
+module SolidusBraintree
+  module Generators
+    class UpdateGenerator < Rails::Generators::Base
+      class_option :auto_run_migrations, type: :boolean, default: false
+      source_root File.expand_path('templates', __dir__)
+
+      def add_migrations
+        run 'bin/rails railties:install:migrations FROM=solidus_braintree'
+      end
+
+      def run_migrations
+        run_migrations = options[:auto_run_migrations] || ['', 'y', 'Y'].include?(ask('Would you like to run the migrations now? [Y/n]')) # rubocop:disable Layout/LineLength
+        if run_migrations
+          run 'bin/rails db:migrate'
+        else
+          puts 'Skipping bin/rails db:migrate, don\'t forget to run it!' # rubocop:disable Rails/Output
+        end
+      end
```

Comment:
Nice, do you think we should double-check that people are not adding `solidus_braintree` and `solidus_paypal_braintree` together in their Gemfile? Not sure this installer is the right place, though.

Reply:
@kennyadsl I would consider this check to be a nice-to-have, since it seems like an edge case. I imagine this can only happen when the developer forgets to remove the `solidus_paypal_braintree` line from the Gemfile. Are there other more common cases that I'm missing?

I'll test it out, to see what the experience for the user.

Reply:
Ouch, this made me think to a (maybe) giant issue. What if someone is currently using both payment methods? Maybe one for credit cart payments and one of paypal payments? I don't think it's that so unrealistic (@spaghetticode do you know any store in that scenario? 🙈). 

Well if that's the case they are done, because they can't add two gems with the same name to the Gemfile. What's the path for them? Do we want to consider it?

Reply:
@kennyadsl The previous edge version of SolidusBraintree had a [warning](https://github.com/solidusio/solidus_braintree/tree/v1.x#deprecation-notice-warning-construction) up for two years that is already deprecated in favor of SolidusPaypalBraintree. Thus, if you are still using SolidusBraintree, you should be aware that there's a high risk in continuing to use it.

However, if you're using both SolidusBraintree and SolidusPaypalBraintree, I'm guessing the path would be 

1. Uninstall the old SolidusBraintree.
2. Replace SolidusPaypalBraintree with the new SolidusBraintree.
3. Either enable Paypal with SolidusBraintree or install SolidusPaypalCommercePlatform.

It seems that it will take a lot more effort to automate that migration path. Whether we want to implement it depends on how many people are still using the old SolidusBraintree.

Your thoughts?

Reply:
@kennyadsl @gsmendoza I think this edge case should just be ignored, anyone in such a dire situation would probably just stay on the currently released versions and call it a day. 

And of course very likely that's not an actual use case.

Reply:
@kennyadsl I intend to ignore for now the edge case of using both payment methods. Do you think we should still consider it?

Reply:
@kennyadsl I have chosen to skip testing the edge case where the user has both payment methods (old SolidusBraintree and SolidusPaypalBraintree), since based on this conversation thread, we think that it's highly unlikely. Let me know if you still have concerns about this.